tests/wake-format/basic/stdout

```diff
@@ -1783,3 +1783,20 @@ def a =
 
     5
 
+def a =
+    a
+    |> b
+    |> c
+
+def a =
+    a
+    |< b
+    |< c
+
+def a =
+    a |> b
+    |< c
+
+def a =
+    a |< b
+    |> c
```

Comment:
need to fix the formatting here

Reply:
It's an interesting formatting bug, definitely!  Nice to have it exercising whatever odd edge case is in the code here.

Reply:
Fixed!